pkgs/applications/office/fantasia-archive/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, fetchzip
+, appimageTools
+, makeDesktopItem
+}:
+
+let
+  pname = "fantasia-archive";
+  version = "0.1.10";
+
+  archive = fetchzip {
```

Comment:
Is there any particular reason we're not building from source?

Reply:
Ah, there isn't. I was already familiar with packaging AppImages so I immediately jumped to that as it was the release format, but in hindsight I probably should have looked into how to directly package it from source

Reply:
If you're interested, you'll want the documentation for [`buildNpmPackage`](https://nixos.org/manual/nixpkgs/stable/#javascript-tool-specific). It looks like this application follows best practices for Electron apps, so it should mostly work out-of-the-box.